aten/src/ATen/native/mkldnn/xpu/detail/Utils.cpp

```diff
@@ -93,7 +93,7 @@ dnnl::memory::data_type get_onednn_dtype_include_double(
 }
 
 bool is_supported_onednn_dtype(const at::Tensor& tensor) {
-  return get_onednn_dtype(tensor, /*allow_undef*/ true) ==
+  return get_onednn_dtype_include_double(tensor) ==
```

Comment:
```suggestion
  return get_onednn_dtype_include_double(tensor) != dnnl::memory::data_type::undef;
```

Reply:
Thanks for suggestions, the code here is modified.